### PR TITLE
[EFR32] Fix building thermostat

### PR DESCRIPTION
**Pull request by jepenven-silabs**
#### Problem
App doesn't build 

#### Change overview
Fix building for wiFi but adding the right sources files that are missing.
In the same time, update the bitmap library to include the thermostat icon since we are now including the LCD files.

#### Testing
Locally on EFR platform

### CODE REVIEW

File: examples/platform/efr32/display/demo-ui-bitmaps.h [hunk 1]
```diff
@@ -264,6 +264,36 @@
         0x00, 0xF8, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,    \
         0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF,    \
         0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF, 0xFF
+#elif IS_DEMO_THERMOSTAT
+#define ON_DEMO_BITMAP                                                                                                             \
```
Comment (andy31415) [inline examples/platform/efr32/display/demo-ui-bitmaps.h:268]: why is this required to fix wifi builds?

The PR description says "fix wifi". Please update PR summary to explain the changes and what they do/why they are needed.

**Review by jmartinez-silabs — APPROVED** [review]

File: examples/platform/efr32/display/demo-ui-bitmaps.h (hunk 1 above)
Comment (jepenven-silabs) [reply examples/platform/efr32/display/demo-ui-bitmaps.h:268]: Since we include the LCD files, might as well include the thermostat icon at the same time. Comment updated

**Review by andy31415 — APPROVED** [review]
